build.gradle

```diff
@@ -16,3 +18,31 @@ dependencyCheck {
         assemblyEnabled = false
     }
 }
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    jacocoAggregation project(':fahrschein'),
+            project(':fahrschein-http-api'),
+            project(':fahrschein-http-simple'),
+            project(':fahrschein-http-apache'),
+            project(':fahrschein-http-spring'),
+            project(':fahrschein-http-jdk11'),
+            project(':fahrschein-inmemory'),
+            project(':fahrschein-metrics-dropwizard'),
+            project(':fahrschein-typeresolver')
+}
+
+reporting {
+    reports {
+        jacocoAggregateReport(JacocoCoverageReport) {
```

Comment:
I like that report name, but changing the name means we need to change the reference in the README.md